### PR TITLE
S3 notify

### PR DESCRIPTION
**Pull request by junghao**
Command line tool as request on #75

### CODE REVIEW

**Comment by gclitheroe** [comment]
Hi Dan,

please could you review this pr as well?

File: cmd/s3-notify/s3-notify.go [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+						break
+					}
+					cnt++
+				}
+			}
+			return !lastPage
+		})
+
+	if err != nil {
+		log.Fatalf("listing s3 objects:", err)
```
Comment (danieldooley) [inline cmd/s3-notify/s3-notify.go]: Not entirely sure but should this be:

```
log.Fatalf("listing s3 objects: %s", err)
```

Comment (junghao) [inline cmd/s3-notify/s3-notify.go]: Sorry my bad. I'll update it.

**Comment by junghao** [comment]
@danieldooley I've updated the log message. Thanks.

**Comment by danieldooley** [comment]
@junghao I can't see anything else wrong, but you might still want Geoff to review it. Probably got a better eye than me.

**Pull request merged**